tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -123,7 +122,10 @@ void InternalCommandsHandler::execute(InternalCommandsHandler::ExecutionRequests
                                 merkleUpdates);
     }
     if (res != OperationResult::SUCCESS) LOG_WARN(m_logger, "Command execution failed!");
-    req.outExecutionStatus = static_cast<uint32_t>(res);
+
+    if (req.outExecutionStatus == static_cast<uint32_t>(OperationResult::UNKNOWN)) {
```

Comment:
Explain that this way we can pass req.outExecutionStatus set to error by the Apollo test and then verify that error reply gets returned.

Reply:
sure